src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,29 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String print(int width) {
+		if (width <= 0 || width % 2 == 0) {
 			return null;
 		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
-			builder.insert(0, chars);
-			builder.append(chars);
+        StringBuilder builder = new StringBuilder();
+        builder.append(generateLine(width, width));
+		for (int currentWidth = width - 2; currentWidth > 0; currentWidth -= 2) {
```

Comment:
"width" is ambiguous. It is also height. And in this line the reader does not understand at once that this is the max size as well. Therefore, it would be better to name the variable something as "diamondWidth" or so